src/HttpRequest.cc

```diff
@@ -738,3 +738,40 @@ HttpRequest::manager(const CbcPointer<ConnStateData> &aMgr, const AccessLogEntry
     }
 }
 
+/// \returns a pointer to known *_port address where the request was accepted
+/// \returns nil for nil requests and when the port address info was not available
+/// cacheIp and localIp are useful when such information may not be available in request
+const Ip::Address *
```

Comment:
missing "static" on return type since this is completely internal to this .cc file.

Reply:
Done